community.html

```diff
@@ -3,15 +3,15 @@
 layout: default
 comment: >-
   Is your sketch missing from the gallery? Please e-mail
-  <a href="mailto:daniel.wheeler@nist.gov">Daniel Wheeler</a> with the
-  following details:
+  <a href="mailto:daniel.wheeler@nist.gov">Daniel Wheeler</a>
+  with the following details:
   <ul class="comment-list">
-    <li>Your name, institutional affiliation, and contact information</li>
-    <li>A professional photograph or preferred avatar</li>
+    <li>Your name, e-mail address, and home page</li>
+    <li>Your professional handles on GitHub and Twitter</li>
```

Comment:
"Your professional handles on GitHub and Twitter or any other social media links that are relevant". Make it clear that we can add links to any social media.

Reply:
At present, [community.html][community.html] only knows about GitHub and Twitter. Can we meet in the middle with this?
> Your professional handles on GitHub and social media (*e.g.* Twitter)

Reply:
Yes, sounds good.